packages/flutter/lib/src/foundation/tree_diagnostics_mixin.dart

```diff
@@ -1686,9 +1817,14 @@ abstract class TreeDiagnostics {
     return toDiagnosticsNode().toStringDeep(prefixLineOne, prefixOtherLines);
   }
 
-  DiagnosticsNode toDiagnosticsNode({ String name, DiagnosticsTreeStyle style });
+  // `name` may be null, but `style` must not be.
+  // TODO(jacobr): document the semantics of the arguments, including what it means when name is null.
+  DiagnosticsNode toDiagnosticsNode({ String name, DiagnosticsTreeStyle style: DiagnosticsTreeStyle.sparse });
 }
 
+// Examples can assume:
```

Comment:
Yes, it's a load-bearing comment. It makes the next line magically declare a class that can then be used in the `## Sample code` sections, which allows me to remove the properties you had in the sample code block.